integration_tests/features/flow_run.feature

```diff
@@ -138,3 +138,14 @@ Feature: `flow run` command
       | zendesk_ticket_data |
     And the script zendesk_ticket_data.check_extra.py output file has the lines:
       | extra_col: yes |
+
+Scenario: fal flow run with target
+  Given the project 001_flow_run_with_selectors
+    When the data is seeded
+
+    When the following command is invoked:
+      """
+      fal flow run --profiles-dir profiles/broken --project-dir $baseDir --select zendesk_ticket_data+ --threads 1 --target custom
+      """
+    Then the following models are calculated:
+      | zendesk_ticket_data |
```

Comment:
Maybe do `fal flow run` without the `--target` to show that it fails. all in the same test

- when data is seeded
- when command fal flow run
- then it raises an exception
- when command fal flow run --target
- then models are calculated